src/views/pages/Account/index.js

```diff
@@ -0,0 +1,90 @@
+import React, {useState, useContext} from 'react';
+import PageTitle from '../../components/PageTitle';
+import Text from '../../components/Text';
+import Button from '../../components/Button';
+import IdentitySubNav from '../../components/IdentitySubNav';
+import Node from '../../../services/Node';
+import {useServiceContext} from '../../../contexts/ServiceContext';
+import {useNavigate} from 'react-router-dom';
+
+function Account() {
+  const [state, setState] = useState({createAccount: true});
```

Comment:
Prefer to use multiple `useState` instead of putting all the state into one object. Multiple advantages to this.

- more efficient targeting of the relevant state for any particular `useEffect` that should update based on the state change
- the state variable name is more expressive and its usage/intent is clear
- state update implementation are simpler and easier to read